pages/samvera/developer_community/contributing_code/review.md

```diff
@@ -64,7 +64,7 @@ The key things to focus on are:
 
 As a reviewer, it's also your responsibility to make sure:
 
-* Does the submitter have a signed CLA [on file](https://wiki.duraspace.org/display/samvera/CLA+submission+list)?
+* Does the submitter have a signed CLA [on file](https://samvera.atlassian.net/wiki/spaces/samvera/pages/405211653/CLA+collection+process)?
```

Comment:
The CLA signing and checking process has just recently changed (or is still in the process of changing) to use `cla-bot`.
```suggestion
* Does the submitter have a signed CLA? (Indicated by a `cla-signed` label in github.)
```